gaphor/diagram/headerlabel.py

```diff
@@ -0,0 +1,8 @@
+from functools import singledispatch
+
+
+def get_diagram_name(diagram):
+    return diagram.name
+
+
+get_header_label = singledispatch(get_diagram_name)
```

Comment:
I would call it `diagram_label`, instead of `get_header_label`.

```
@singledispatch
diagram_label(diagram: Diagram):
    return diagram.name
```

Reply:
> (Note to self: get_icon_name should be renamed to icon_name)

I might as well do that myself with a following commit.

Reply:
Thanks!